src/ProtobufInterfaces/proto/point.proto

```diff
@@ -28,9 +28,12 @@ enum PreventedPosition {
 
 service Point {
     rpc PreventEndPosition (SimulatedPositionMessage) returns (google.protobuf.Empty) {}
+    rpc PutIntoUnintendedPosition (SimulatedPositionMessage) returns (google.protobuf.Empty) {}
```

Comment:
As discussed, Unintended position is a B4R2 word. Suggestion: Add second PutIntoTrailedPosition method, and throw an exception if we're simulating a baseline where this wording doesn't apply